src/main/java/rcms/utilities/daqexpert/persistence/LogicModuleRegistry.java

```diff
@@ -89,6 +90,8 @@
 
 	FEROLFifoStuck		   		(new FEROLFifoStuck(),		       ConditionGroup.OTHER,                 "",                                                  500,  10500),
 
+	RuFailed                (new RuFailed(),                 ConditionGroup.OTHER,                 "",                                                  900,   9500),
```

Comment:
Is there any particular reason for runOrder set to 900 instead of next free number which is 36?